src/Interpreters/InternalTextLogsQueue.cpp

```diff
@@ -46,24 +46,26 @@ void InternalTextLogsQueue::pushBlock(Block && log_block)
         LOG_WARNING(&Poco::Logger::get("InternalTextLogsQueue"), "Log block have different structure");
 }
 
-const char * InternalTextLogsQueue::getPriorityName(int priority)
+std::string_view InternalTextLogsQueue::getPriorityName(int priority)
 {
-    /// See Poco::Message::Priority
+    using namespace std::literals;
 
-    static constexpr const char * const PRIORITIES[] =
+    /// See Poco::Message::Priority
+    /// NOTE: std::make_array() is experimental now, hence we need to pass the size.
+    static constexpr const std::array<std::string_view, 10> PRIORITIES =
```

Comment:
```suggestion
    static constexpr std::array PRIORITIES =
```
did you try this?

Reply:
Interesting, didn't know that `array<T>` is a special case:

https://github.com/llvm/llvm-project/blob/4adf89fc65f6ffda8add795d002334b6bbea55f5/libcxx/include/array#L253

This indeed works. Thanks.

Reply:
It's because of https://en.cppreference.com/w/cpp/language/class_template_argument_deduction
But you can't do partial CTAD yet so you have to somehow force it to be of certain type before.

Reply:
Yeah, I know about CTAD, but at first I use `std::array<string_view> = { "foo" }` to convert "foo" to `string_view`, so I have to specify the type, but didn't know that `array<T>` is a special "empty" array.